workshop/machine-learning-autoai/README.md

```diff
@@ -0,0 +1,275 @@
+# Machine Learning with AutoAI
+
+For this part of the workshop, we'll learn how to use [AutoAI](https://www.ibm.com/support/producthub/icpdata/docs/content/SSQNUZ_current/wsj/analyze-data/autoai-overview.html).
+AutoAI is a capability that automates machine learning tasks to ease the tasks of data scientists. It automatically prepares your data for modeling, chooses the best algorithm for your problem, and creates pipelines for the trained models.
+
+This section is broken up into the following steps:
+
+1. [Upload data to project](#1-upload-the-dataset)
+2. [Set up your AutoAI environment and generate pipelines](#2-set-up-your-autoai-environment-and-generate-pipelines)
+3. [Save AutoAI Model](#3-save-autoai-model)
+4. [Deploy and test the model](#4-deploy-and-test-the-model)
+
+>*Note: The lab instructions below assume you have a project already. If not, follow the instructions in the pre-work section to create a project.*
+
+## 1. Upload the Dataset
+
+* We will upload the dataset used to train our model. If you have not yet downloaded or cloned Git Repo as outlined in the pre-work section. You can download/save the [german-credit-data.csv](https://raw.githubusercontent.com/IBM/credit-risk-workshop-cpd/master/data/original/german_credit_data.csv) directly to your machine.
+
+* From the project overview page, click on the `Assets` tab to open the assets page where your project assets are stored and organized. You can load a local dataset using the interface on the right.
+
+* Click `browse` to find and upload the [german-credit-data.csv](../dataset/Telco-Customer-Churn.csv) dataset from your machine:
```

Comment:
This link should be `https://github.com/IBM/credit-risk-workshop-cpd/blob/master/data/original/german_credit_data.csv` for the online link.
or could be `../../data/german_credit_data.csv` for local.